src/dmd/expression.d

```diff
@@ -2296,7 +2296,7 @@ extern (C++) final class StringExp : Expression
      * Returns:
      *      number of code units
      */
-    final size_t numberOfCodeUnits(int tynto = 0) const
+    size_t numberOfCodeUnits(int tynto = 0) const
```

Comment:
Why is this required?

Reply:
the class is final. So are its member funcs.

Reply:
to be perfectly clear: the useless_final_attribute check didn't pass.